packages/docsearch-react/src/SearchBox.tsx

```diff
@@ -23,9 +30,16 @@ interface SearchBoxProps
   inputRef: MutableRefObject<HTMLInputElement | null>;
   onClose: () => void;
   isFromSelection: boolean;
+  translations?: SearchBoxTranslations;
 }
 
-export function SearchBox(props: SearchBoxProps) {
+export function SearchBox({ translations = {}, ...props }: SearchBoxProps) {
+  const {
+    resetButtonTitle = 'Clear the query',
+    resetButtonAriaLabel = 'Clear the query',
+    cancelButtonText = 'Cancel',
+    cancelButtonAriaLabel = 'Cancel',
```

Comment:
I think it's still good to have a starting point, this could also potentially give user the opportunity to contribute if they want to help us improve the accessibility, wdyt?

Reply:
I don't know much about that topic unfortunately, I just know that it's not as easy as it seems and it might give the wrong idea that others props might be changeable. (or maybe aria should not even be translated)
I would just note that in open issue if we want to give users the opportunity to help